src/DoctrineAuditBundle/AuditConfiguration.php

```diff
@@ -66,19 +68,26 @@ class AuditConfiguration
      */
     private $annotationLoader;
 
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $dispatcher;
+
     public function __construct(
         array $config,
         UserProviderInterface $userProvider,
         RequestStack $requestStack,
         FirewallMap $firewallMap,
         EntityManagerInterface $entityManager,
-        ?AnnotationLoader $annotationLoader = null
+        ?AnnotationLoader $annotationLoader,
+        EventDispatcherInterface $dispatcher
     ) {
         $this->userProvider = $userProvider;
         $this->requestStack = $requestStack;
         $this->firewallMap = $firewallMap;
         $this->entityManager = $entityManager;
         $this->annotationLoader = $annotationLoader;
+        $this->dispatcher = LegacyEventDispatcherProxy::decorate($dispatcher);
```

Comment:
`LegacyEventDispatcherProxy` has been introduced in 4.3. So, it will break all the lower apps

Reply:
good point